### PR TITLE
Setup react and linters

### PR DESCRIPTION
**Pull request by vanillainanutshell**
# What was done in this PR:

- Set React
- Set Linters

### CODE REVIEW

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Matt, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project without a readme file, please take [the template ](https://github.com/microverseinc/readme-template/blob/master/README.md)provided by Microverse and adjust it so it suits you and your project.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by omar-labana — CHANGES_REQUESTED** [review]
Hi @vanillainanutshell, 

Excellent work so far and nice GIF! 👌
You have implemented almost all the required features. ✔️ 
There are some cases that you need to adjust but you are almost done! 😀

## Required Changes ♻️
- [ ] I can see that you managed to use the `create-react-app` command successfully but you have invoked it into a sub directory called `my-app` not the root directory. There are two ways to solve this, the easiest is to move all the files but the README from my-app to the root directory or to delete the my-app directory and invoke the create-react-acpp command replacing `my-app` with a `.`

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
_N/A_

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -1 +1,113 @@
...
+- CSS
+- JavaScript
+- React
+
+
+### Setup
+
+- Clone this repo URL "https://github.com/vanillainanutshell/math-magician"
+
+## Authors
```
Comment (rloterh) [inline README.md:18]: - [ ] OPTIONAL
To improve the professionalism aspect of your ReadMe, kindly change the header title _Authors_ to _Author_, as this project is done solely.

File: README.md [hunk 2]
```diff
@@ -1 +1,113 @@
...
+
+👤 **Author**
```
Comment (rloterh) [inline README.md:20]: - [ ] OPTIONAL
Kindly replace the text **Author** with your full name, so as to make your ReadMe more professional

File: README.md [hunk 3]
```diff
@@ -1 +1,113 @@
...
+Builds the app for production to the `build` folder.\
+It correctly bundles React in production mode and optimizes the build for the best performance.
+
+The build is minified and the filenames include the hashes.\
+Your app is ready to be deployed!
+
+See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+
+### `npm run eject`
+
```
Comment (rloterh) [inline README.md:76]: - [ ] OPTIONAL
Kindly ensure to remove all unnecessary details from your ReadMe so as to keep it cleaner.

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Matt, 

Your project is complete! Great job for making the necessary changes recommended in the previous review. There is nothing else to say other than... it's time to merge it 🍾 :shipit:. Congratulations! 🎉🥂

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Kindly consider my comment(s) marked **[OPTIONAL]** under this review._
<br>
Cheers and Happy coding!👏 💻 🍷 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 1 more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**